github/ci/services/ci-search/patches/statefulset.yaml

```diff
@@ -28,6 +28,8 @@ spec:
         - --bugzilla-url=https://bugzilla.redhat.com/rest
         - --bugzilla-search=OPEN version:4.7,4.6,4.5,4.4,4.3,4.2 product="Container Native Virtualization (CNV)" AND delta_ts>-2w
         - --bugzilla-token-file=/etc/bugzilla/api
+      securityContext:
+        fsGroup: 1000
```

Comment:
That does not look like running as root.

Reply:
Indeed, running as root was the first approach, but also works just by setting the file system group and letting it to run as user 1000, I've already updated the PR title and commit message.